src/components/ListMapStatic/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react"
+import styled from "styled-components"
+
+const Outer = styled.section`
+  height: 450px;
+  border: 1px solid ${props => props.theme.styles.cardShadow};
+  margin-bottom: 25px;
+  .list-map {
+    height: 100%;
+  }
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  img {
+    flex-shrink: 0;
+    min-width: 100%;
+    min-height: 100%;
+  }
+`
+
+export default ({ results }) => {
+  let resultsLatLng = []
+  results.map(result =>
+    result.locations.map(location => {
+      if (
+        location.geometry.coordinates[0] &&
+        location.geometry.coordinates[1]
+      ) {
+        resultsLatLng.push([
+          `${location.geometry.coordinates[1]},${location.geometry.coordinates[0]}`,
+        ])
+      }
+      return null
+    })
+  )
+
+  // without access to googles LatLngBounds we're attempting to work out the central point of coords here
+  // https://stackoverflow.com/questions/15719951/auto-center-map-with-multiple-markers-in-google-maps-api-v3
+
+  const coordsAsString = resultsLatLng.join("|")
+  const filteredtextCoordinatesArray = coordsAsString.split("|")
+
+  let centerLatArray = []
+  let centerLngArray = []
+
+  filteredtextCoordinatesArray.map(centerCoords => {
+    let centerCoordsArray = centerCoords.split(",")
+    if (!isNaN(Number(centerCoordsArray[0]))) {
+      centerLatArray.push(Number(centerCoordsArray[0]))
+    }
+
+    if (!isNaN(Number(centerCoordsArray[1]))) {
+      centerLngArray.push(Number(centerCoordsArray[1]))
+    }
+    return null
+  })
+
+  var centerLatSum = centerLatArray.reduce(function (a, b) {
```

Comment:
I tried pulling this down and running it but I get this error when I try to view the map: `TypeError: reduce of empty array with no initial value` which looks like it's coming from this line and line 62.

I don't have any locations in my local Outpost version so that's probably why, but would still be good to not get an error!

Reply:
I have added a guard against this, it just means the map will be centered over the sea 0,0  instead of in the middle of all the pins on the static image but withouth the locations it wont have any pins anyway!